cluster/gce/util.sh

```diff
@@ -665,8 +665,18 @@ function kube-up {
   echo "  up."
   echo
 
+  # curl in mavericks is borked.
+  secure=""
+  if which sw_vers > /dev/null; then
+    if [[ $(sw_vers | grep ProductVersion | awk '{print $2}') = "10.9."* ]]; then
```

Comment:
`sw_vers -productVersion` gives you the number directly without needing grep/awk. At least it does on Yosemite. Maybe Mavericks isn't advanced enough.... :)